packages/react-dom/src/test-utils/ReactTestUtils.js

```diff
@@ -42,7 +42,9 @@ const [
   runEventsInBatch,
 ] = ReactDOM.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED.Events;
 
-function Event(suffix) {}
+function Event(type) {
+  this.type = type;
```

Comment:
I‘m a bit confused about this - where do we use this event constructor?

Reply:
SimulateNativeEvent:

https://github.com/facebook/react/blob/f6fb03edffcc5918ad2f37ed45bdc7c0c3b3d199/packages/react-dom/src/test-utils/ReactTestUtils.js#L479-L498